app/settings/settings.css

```diff
@@ -97,6 +97,14 @@ input,
   font-weight: 400;
   font-family: -apple-system, system-ui, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
   font-size: 16px;
+  position: relative;
+  z-index: 15;
+}
+
+.profiles-container archipelago-profiles .profile-list archipelago-profile .profile-remove {
```

Comment:
I would prefer this to be on the left side, in the margin and only visible when you hover over the profile.

Reply:
Moved to left and added hover state @npezza93

Reply:
Moved to left and added hover state